pages/mesosphere/dcos/services/dse/3.1.0-6.7.5/release-notes/index.md

```diff
@@ -1,13 +1,44 @@
 ---
 layout: layout.pug
 navigationTitle: Release Notes
-excerpt: Release Notes for DSE v. 3.0.0-6.7.2
+excerpt: Release Notes for DSE v. 3.1.0-6.7.5
 title: Release Notes
 menuWeight: 10
 model: /mesosphere/dcos/services/dse/data.yml
 render: mustache
 ---
 
+# Version 3.1.0-6.7.5
+
+## Updates
+- Upgraded DSE to version `6.7.5` and OpsCenter to version `6.7.4`
+- Upgraded SDK to `0.57.1`. For more information see SDK's Release Notes for: 
+  - [0.57.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1)
+  - [0.57.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0)
+  - [0.56.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.3)
+  - [0.56.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.2)
+  - [0.56.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.1)
+  - [0.56.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.0)
+  - [0.55.5](https://github.com/mesosphere/dcos-commons/releases/tag/0.55.5)
+
+## New Features
+- Added support for DSS volume profiles
```

Comment:
Let's add a section to the docs similar to what we did with Cassandra and then add a link to it

Reply:
Do you mean you want to link "DSS volume profiles" to something like this?

https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/volume-plugins/

Reply:
@SarahStegall Yes i will add that link and related brief info.

Reply:
Thanks. Let me know when you've done so.